homeassistant/components/media_player/gpmdp.py

```diff
@@ -0,0 +1,115 @@
+"""
+Support for Google Play Music Desktop Player.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.gpm_dp/
+"""
+
+from homeassistant.components.media_player import (
+    MEDIA_TYPE_MUSIC, SUPPORT_NEXT_TRACK, SUPPORT_PREVIOUS_TRACK,
+    SUPPORT_PAUSE, MediaPlayerDevice)
+from homeassistant.const import (
+    STATE_PLAYING, STATE_PAUSED, STATE_OFF)
+
+REQUIREMENTS = ['websocket-client==0.35.0']
```

Comment:
`requirements_all.txt` file is not up to date. Run `script/gen_requirements_all.py` and include `requirements_all.txt` in your PR.